src/assets/locales/fr.json

```diff
@@ -508,7 +570,7 @@
   "sign-in-modal.title": "Se connecter",
   "sign-in-modal.wait-for-verification": "<p>Email envoyé à {email} !</p><p>Pour continuer, cliquez sur le lien dans l'email en utilisant votre téléphone, tablette ou PC.</p><p>Aucun email ? Vous ne pourrez pas créer de compte.</p>",
   "spectating-label.label": "Observer",
-  "support.copied": "copié!",
+  "support.copied": "copié !",
```

Comment:
Should there be a space here? `"copié!"` `"copié !"`

Reply:
Yes! In French you put a space before "!", "?", ":", ";".

Reply:
> "preferences-screen.preference.material-quality-setting": "Qualité des textures",
>This refers to materials/textures. e.g Low quality will not display metallic, roughness and normal maps.
Ok so I think translating it by textures here is good. And this is generally the translation you see in video games.

Reply:
Thanks for the clarification, this jumped out to me because the original French translation didn't have the space. lgtm.